content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/tutorial-k8s.md

```diff
@@ -1,227 +1,282 @@
+```yaml
+workers:
+  rpc:
+    service:
+      type: LoadBalancer
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-internal: true
+        service.beta.kubernetes.io/aws-load-balancer-scheme: internal
+        service.beta.kubernetes.io/aws-load-balancer-type: nlb
+        external-dns.alpha.kubernetes.io/hostname: rpc-worker.dev1.corda.cloud
 ```
-    {{% /tab %}}
-    {{< /tabs >}}
-
-2. Clone the [corda/corda-cli-plugin-host](https://github.com/corda/corda-cli-plugin-host) repository:
-
-    ```sh
-    git clone https://github.com/corda/corda-cli-plugin-host.git
-    git -C corda-cli-plugin-host checkout release/version-1.0.0-DevPreview2
-    ```
-
-3. Clone the [corda/corda-api](https://github.com/corda/corda-api) repository:
-
-    ```sh
-    git clone https://github.com/corda/corda-api.git
-    git -C corda-api checkout release/os/5.0-DevPreview2
-    ```
-
-4. Clone the [corda/corda-runtime-os](https://github.com/corda/corda-runtime-os) repository:
-
-    ```sh
-    git clone https://github.com/corda/corda-runtime-os.git
-    git -C corda-runtime-os checkout release/os/5.0-DevPreview2
-    ```
-
-5. Build all of the Corda Docker images with Gradle in the `corda-runtime-os` repository:
-
-    ```sh
-    cd corda-runtime-os
-    ./gradlew clean publishOSGiImage -PcompositeBuild=true
-    ```
-
-## Install Corda
-
-There is a `values.yaml` file at the root of the `corda-runtime-os` repository that overrides the default values in the Corda Helm chart.
-These values configure the chart to use the images you just built and specify the location of the Kafka and PostgreSQL instances created by the `corda-dev` Helm chart.
-They also set the initial admin user password to `admin`.
-
-1. Install the chart as follows by running from the root of the `corda-runtime-os` repository:
-
-   ```sh
-   helm install corda -n corda charts/corda --values values.yaml --wait
-   ```
-
-When the commmand completes, the RPC endpoint should be ready to access.
-
-### Troubleshooting
-
-If the install times out, it indicates that not all of the worker pods reached ready state.
-Use the following command to list the pods and their current state:
 
-```sh
-kubectl get pods -n corda
+### PostgreSQL
+
+The password for PostgreSQL can be specified directly as a Helm override but this is not recommended. Instead, create a Kubernetes secret containing the password with a key of password. By default, the install expects a database called `cordacluster` but this can be overridden. You can then define the PostgreSQL configuration as follows:
```

Comment:
```suggestion
The password for PostgreSQL can be specified directly as a Helm override but this is not recommended. Instead, create a Kubernetes secret containing the password with a key of `password`. By default, the install expects a database called `cordacluster` but this can be overridden. You can then define the PostgreSQL configuration as follows:
```